functions/UpdateBookingSlot-BookToISV/run.csx

```diff
@@ -1,4 +1,8 @@
 ﻿#load "..\Shared\httpUtils.csx"
+#load "..\Shared\CompleteAppointmentDTO.csx"
+#load "..\Shared\TechnicalResource.csx"
+#load "..\Shared\EncryptionUtils.csx"
+#load "..\Shared\isv.csx"
```

Comment:
Apologies, I didn't know this didn't work, it worked locally. As far as nanoservices are concerned, then I would have thought that reuse was more important than having all the code in a single file. I wanted to reuse the logic in 'UpdateBookingSlot-BookToISV' - which actually does 5 separate db calls and uses 3 different data structures - not exactly a nano-service! :-) I needed the bulk of it's capabilities but not necessarily with the same conditions. So I needed to re-use the data structures and queries. In hindsight - maybe should have broken the existing service into parts and then called the services based on the conditions I needed - we should chat